src/components/clusterConfiguration/BaremetalInventory.tsx

```diff
@@ -1,19 +1,30 @@
 import React from 'react';
 import { Text, TextContent, Button } from '@patternfly/react-core';
 import HostsTable from '../hosts/HostsTable';
-import { Cluster } from '../../api/types';
+import { Cluster, HostRequirements } from '../../api/types';
 import { DiscoveryImageModalButton } from './discoveryImageModal';
 import {
   HostsNotShowingLink,
   DiscoveryTroubleshootingModal,
 } from './DiscoveryTroubleshootingModal';
+import { getHostRequirements } from '../../api/hostRequirements';
+import { fetchApiEndpoint } from '../../api';
 
 interface BareMetalInventoryProps {
   cluster: Cluster;
 }
 
 const BaremetalInventory: React.FC<BareMetalInventoryProps> = ({ cluster }) => {
   const [isDiscoveryHintModalOpen, setDiscoveryHintModalOpen] = React.useState(false);
+  const [hostRequirements, setHostRequirements] = React.useState<HostRequirements | null>(null);
+
+  React.useEffect(() => {
```

Comment:
I was thinking about placing this fetch at higher level in the hierarchy but so far there is no other use for it.